composer.json

```diff
@@ -5,7 +5,7 @@
     "license": "proprietary",
     "require": {
         "php": ">=7.2",
-        "punchout-catalogs/punchout-catalog-spryker": "^2.0.0",
+        "punchout-catalogs/punchout-catalog-spryker": "^2.2.0",
```

Comment:
what is the reason to increase this dependency version?
in case we do not use anything introduced exactly at `2.2.0` this should be reverted

Reply:
Actually, the `disabled` option handling added exactly at `2.2.0` .